### ISSUE TITLE
Paths resolved incorrectly for static files depending on destination

### ISSUE THREAD

**Issue by BlackWolf** [jsdoc3/jsdoc#939]
When I use the new static files options (see #393) the files are somtimes copied to the wrong location, depending on the argument given to the `-d` parameter on the command line.

The issue is that jsdoc seems to use the value of `-d` as the root for relative paths. So if my current directory is not the disk's root, but I pass an absolute path to `-d`, jsdoc will copy files to the wrong location:

### Example that produces erronous results

**jsdoc.json**:

```
{
  "plugins": [],
  "templates": {
    "default": {
      "staticFiles": {
        "include": [ "./static" ]
      }
    }
  }
}
```

**command line call:**

```
cd ~/Projects/WebProject
jsdoc  /path/to/sources/*.js -c ./jsdoc.json -d ~/Projects/WebProject/docs
```

The static files will be copied to `~/Projects/WebProject/~/Projects/WebProject/docs/` in this case.

### Example that produces correct results

**jsdoc.json**:
[same as before]

**command line call:**

```
cd ~/Projects/WebProject
jsdoc  /path/to/sources/*.js -c ./jsdoc.json -d ./docs
```

**Comment by hegemonic** [comment]
@BlackWolf If your example is correct, the bug is not about relative paths per se--it occurs when you use a tilde (`~`) in the `-d` option. Does that sound correct? If so, I'll investigate; if not, perhaps you can clarify.

**Comment by BlackWolf** [comment]
Unfortunately the problem seems to occur anytime I pass an absolute path to `-d` (and my current directory is not `/`). I just used the tilde to keep the example short. The same thing would happen for

```
cd /Users/BlackWolf/Projects/WebProject
jsdoc /path/to/sources/*.js -c ./jsdoc.json -d /Users/BlackWolf/Projects/WebProject/docs
```

In this case, a folder `/Users/BlackWolf/Projects/WebProject/Users/BlackWolf/Projects/WebProject/docs` is created and the static files are copies there.

**Comment by hegemonic** [comment]
Thanks for clarifying! I'll investigate.

**Comment by hegemonic** [comment]
Okay, this issue should be fixed on master and releases/3.3. These filepath issues can be tricky, so if you're still seeing a problem, please let me know!
